events/wc_frostwolf_events.txt

```diff
@@ -879,29 +939,30 @@ province_event = {
 		d_frostwolf = {
 			holder_scope = {
 				if = {
-					limit = { frostwolf_exodus_d_frostwolf_trigger = yes }
-					narrative_event = { id = WCFW.20 }
+					limit = { frostwolf_leader_exists = yes }
+					narrative_event = { id = WCFW.40 }
 				}
 				else = {
-					character_event = { id = WCFW.200 }
+					73 = { province_event = { id = WCFW.600 } }
 				}
 			}
 		}
 	}
 }
+
 # Delayed event
 narrative_event = {
-	id = WCFW.20
-	title = EVTTITLE_WCFW_20 #We are ready.
-	desc = EVTDESC_WCFW.20
+	id = WCFW.40
+	title = EVTTITLE_WCFW_40
+	desc = EVTDESC_WCFW_40 # Our clan is ready to leave.
 	picture = GFX_evt_campfire
 	border = GFX_event_narrative_frame_war
 
 	is_triggered_only = yes
 
 	option = {
-		name = EVTOPTA_WCFW.20 #Let's hit the road!
-		custom_tooltip = { text = EVTOPTA_WCFW_20_tooltip }
+		name = EVTOPTA_WCFW_40 # Let us leave this place!
+		custom_tooltip = { text = EVTOPTA_WCFW_40_tooltip } # We're going south.
 		73 = {
 			province_event = {
 				id = WCFW.121
```

Comment:
Went south and stole Lordaeronian ship.
<details>
<summary>Click to expand</summary>

![image](https://user-images.githubusercontent.com/46384992/67239645-b1e31d80-f460-11e9-9874-b241213b5542.png)

</details>

Reply:
Stupid typo, that should go to `WCFW.141` instead of `WCFW.121`. Resolved.